tests/test_spherePoint.py

```diff
@@ -275,17 +288,26 @@ def testTicket1761(self):
     def testAtPoleValue(self):
         """Test if atPole() returns the expected value.
         """
-        poleList = [SpherePoint(42.0*degrees, lat) for lat in self._poleLatitudes] + \
-            [SpherePoint(42.0*degrees, 0.0*radians - lat) for lat in self._poleLatitudes] + [
-            SpherePoint(afwGeom.Point3D(0.0, 0.0, 1.0)),
-            SpherePoint(afwGeom.Point3D(0.0, 0.0, -1.0)),
-        ]
-        nonPoleList = [SpherePoint(42.0*degrees, self.nextDown(lat)) for lat in self._poleLatitudes] + \
-            [SpherePoint(42.0*degrees, self.nextUp(0.0*radians - lat)) for lat in self._poleLatitudes] + [
-            SpherePoint(afwGeom.Point3D(9.9e-7, 0.0, 1.0)),
-            SpherePoint(afwGeom.Point3D(9.9e-7, 0.0, -1.0)),
-            SpherePoint(0.0*degrees, nan*degrees),
-        ]
+        poleList = \
+            [SpherePoint(42.0*degrees, lat) for lat in self._poleLatitudes] + \
+            [SpherePoint(42.0, lat.asDegrees(), degrees) for lat in self._poleLatitudes] + \
+            [SpherePoint(42.0*degrees, 0.0*radians - lat) for lat in self._poleLatitudes] + \
+            [SpherePoint(42.0, (0.0*radians - lat).asDegrees(), degrees) for lat in self._poleLatitudes] + \
```

Comment:
`-lat.asDegrees()` would be more concise.

Actually, since I added a negation operator to `Angle`, all the references to `0.0*radians` can probably be removed.